src/usecases/associate-entity-use-case.ts

```diff
@@ -20,58 +24,72 @@ export type AssociateEntityUseCaseParams = {
 };
 
 export const associateEntityUseCase = {
+	/**
+	 * @throw {ForbiddenByFigmaUseCaseError} Not authorized to access Figma.
+	 * @throw {Error} Unknown error.
+	 */
 	execute: async ({
 		entity,
 		associateWith,
 		atlassianUserId,
 		connectInstallation,
 	}: AssociateEntityUseCaseParams): Promise<AtlassianDesign> => {
-		const figmaDesignId = FigmaDesignIdentifier.fromFigmaDesignUrl(entity.url);
```

Comment:
**note:** The PR add only `try` / `catch` and error translation in all the use case -- there is no logical change.